### PR TITLE
React home not found

### PR DESCRIPTION
**Pull request by Varelazi**

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
 const Home = () => {
     return (
         <>
-          <h1>Home Page</h1>
-        </>
+      <div className= "homecontainer"> 
+        <h2 className= "introtext" >Find your new apartment here at Twilight Towers!</h2>
+        <img src={logo} alt="logo for website" id="apartment-logo"></img>
+      </div>
```
Comment (SunkissedQueen) [inline src/pages/Home.js]: indentation

File: src/pages/NotFound.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React from "react"
+import notfound from "../assets/404error.jpeg"
 
 const NotFound = () => {
     return (
         <>
-          <h1>NotFound Page</h1>
+        <img src={notfound} alt="404 image" className= "error"></img>
```
Comment (SunkissedQueen) [inline src/pages/NotFound.js]: indentation

File: src/_test_/NotFound.test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+describe("<NotFound />", () => {
+    it("renders an image on the not found page", () => {
+        render(
+            <BrowserRouter>
+            <NotFound />
+            </BrowserRouter>
+        )
+     screen.debug()
+     screen.logTestingPlaygroundURL()
```
Comment (SunkissedQueen) [inline src/_test_/NotFound.test.js]: debugging tools

File: src/_test_/Home.test.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import Home from '../pages/Home'
+
+describe("<Home />", () => {
+    it("renders a introduction", () => {
+        render(
+            <BrowserRouter>
+            <Home/>
+            </BrowserRouter>
+        )
+        screen.logTestingPlaygroundURL()
```
Comment (SunkissedQueen) [inline src/_test_/Home.test.js]: debugging tool

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Please see comments. Just a noted: decide as a team if you all want 4 spaces or 2 spaces on the indentation.

**Review by sjproctor — APPROVED** [review]

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**